[2025-08-18 15:30:28,942] {taskinstance.py:1153} INFO - Dependencies not met for <TaskInstance: send_weather_to_kafka.stream_hdfs scheduled__2025-08-18T13:22:48.013844+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-08-18 15:30:28,946] {local_task_job.py:101} INFO - Task is not able to be run
